diagnostic.md

```diff
@@ -6,76 +6,85 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```bash
-// your response here
+specify your route with to:/books/4 in routes
+or write a CRUD request specifically into terminal using CURL request
 ```
 
 In what file does the above code go?
 
 ```bash
-// your response here
+routes.rb
+
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```bash
-// your response here
+It brings the current table data into the table that was just created
+
 ```
 
 What do we call the string rendered from our server?
 
 ```bash
-// your response here
+The error message?  I guess.  The 'do next to make this thing work' message?
+
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```bash
-// your response here
+scaffold will set up an entire API for you, with routes, controllers, etc.  Controller will just set up the Controller for you.
 ```
 
 What is the job of the `serializer`?
 
 ```bash
-// your response here
+To filter out things that you don't want seen.
+
 ```
 
 What do we expect the command `Patient.all` to return?
 
 ```bash
-// your response here
+All of the patients.
+
 ```
 
 We use the constant `Patient` in the `PatientsController`, but where is it
 actually defined?
 
 ```bash
-// your response here
+Hmm, so the parameters of the Patient are defined in controller, but the Class is actually
+declared in the Model, so...both?  I guess really it's defined in the table, because the columns associated
+with the patient are what define it.
+
```

Comment:
Yes, basically it's defined in the model, but all of it's methods are in the controller. Does that make sense?